woocommerce/woocommerce-gutenberg-products-block
# MiniCart: Check that data is countable before counting

## What

Use `is_countable()` before using `count()` on data in the Mini Cart block.

Fixes #

## Why



Rector has reported issues with using `count()` in places there could be potentially non-countable data which can result in a fatal error.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Add Mini Cart to header of your site
2. Regression test the Mini Cart and ensure there are no regressions or fatal errors as a result of this change.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
